### PR TITLE
donations: copy content from https://wiki.haskell.org/Donate_to_Haskell.org

### PR DESCRIPTION
**Pull request by ketzacoatl**
This contributes to #59, moving authoritative content from the wiki to the primary haskell.org website.

We can update with commits to clean it up, but ATM this is the content from that page, more or less as-is. 

After this is cleaned up and live, we can then update the wiki page to replace its content with a link back to this donations page on haskell.org.

### CODE REVIEW

File: donations.markdown
Comment (rebeccaskinner) [inline donations.markdown:49]: This paragraph seems to have some redundancy with the paragraph following it. Maybe we could merge them into a single paragraph?

File: donations.markdown
Comment (rebeccaskinner) [inline donations.markdown:36]: Could we think about re-wording the first sentence here? Maybe the mention of volunteering should go in a separate section that has some information on how contact the committee to volunteer? It seems odd to me to open the paragraph on donations by instead suggesting that someone volunteer.

**Review by rebeccaskinner — CHANGES_REQUESTED** [review]
Thanks for the PR. I had a couple of notes that I think could help improve this if you are up for making some edits.

File: donations.markdown
Comment (ketzacoatl) [reply donations.markdown:49]: Yea, I have pretty much copied the content from the wiki. Feel free to add a commit or GitHub change suggestion with the wordsmithing you feel is best.

File: donations.markdown
Comment (ketzacoatl) [reply donations.markdown:36]: I don't disagree, but I didn't want to judge the wiki content alone and before copying it.. I wanted to copy it over as-is, and then let us work collaboratively to clean it up.

**Comment by ketzacoatl** [comment]
I went to edit/update the wiki page at https://wiki.haskell.org/Donate_to_Haskell.org, to point it towards https://www.haskell.org/donations/ and this PR, but that page is not editable by regular contributors, so we are again blocked on wiki admins who have not been very helpful in making these updates (just as with #64). This process is tiresome. No wonder the wiki is so poorly maintained!

**Comment by gbaz** [comment]
Those pages, as I explained, were protected not because we wanted to, but because others complained they could not trust their "officialness" if not locked. Any haskell.org committee member has always been able to request permissions to their account to edit such pages. Nobody has explicitly contacted the admins to get such permissions. Emily has a wiki account and I made sure she got permissions the other day, but I imagine she is quite busy. 

Since pages now on the haskell.org website now exist and supersede the older wiki ones, I have now unlocked the wiki pages. Hopefully that should help in sorting this out.

**Comment by ketzacoatl** [comment]
At least with respect to helping me contribute meaningful updates to the community infrastructure, your actions to help and unblock are much more valuable, useful, and appreciated than your explanations of why something is the way that it is.

Thank you for unblocking those two pages so we can complete this task.

**Comment by ketzacoatl** [comment]
I have updated https://wiki.haskell.org/Donate_to_Haskell.org to point to both https://www.haskell.org/donations/ and this PR. When the PR is merged, we can either drop the PR link from the wiki article, or leave it if people think there's value in doing so.

**Comment by TikhonJelvis** [comment]
@rebeccaskinner Could you review this PR again? Thanks!

**Review by rebeccaskinner — APPROVED** [review]
Going to go ahead and approve for now to unblock the work and then we can take a second pass to clean up the wording late. Thanks for all the work on the contribution!

File: donations.markdown
Comment (tomjaguarpaw) [inline donations.markdown:45]: The formatting here seems off. Shouldn't `How to donate` and `Contact` be headings?

File: donations.markdown
Comment (tomjaguarpaw) [inline donations.markdown:51]: Given the new text the old sentence "Haskell.org, Inc. is a non-profit registered in New York, dedicated to promoting educational and scientific progress around the Haskell programming language." seems redundant. Should it be removed?

**Review by tomjaguarpaw — CHANGES_REQUESTED** [review]
Please address at least headings formatting. I'm flexible on whether removing the now-redundant sentence should be left for a second PR.

File: donations.markdown
Comment (ketzacoatl) [reply donations.markdown:46]: I assume the committee, or other contributors can clean that up. I did not want to get bogged down in wordsmithing.. my goal is primarily to ensure there is one canonical page for this type of authoritative content (and merging what's on the wiki is a first step). Feel free to add commits in a new PR, or to suggest some specific change in this PR.

Comment (tomjaguarpaw) [reply donations.markdown:46]: That's fair enough.

**Review by tomjaguarpaw — APPROVED** [review]
This PR strictly improves the status quo so I suggest it be merged.

There's more to do after this so let's try to move quickly and maintain momentum.

**Pull request merged**
